2001-3000/LIT2085OntheL.xml

```diff
@@ -63,6 +78,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:CLP46i_may_form_part_of" active="LIT2085OntheL" passive="LIT1493Gadlas"/>
                <relation name="saws:isAttributedToAuthor" active="LIT2085OntheL" passive="PRS5720JohnChr"/>
```

Comment:
Perhaps also mark as different from LIT2093OnHero, also let me repeat my suggestion to use Proverbio's book as clavis for Chrysostomian homilies

Reply:
I will get back to this, thank you